### PR TITLE
[core] AptosConnect Telegram support

### PR DESCRIPTION
**Pull request by hardsetting**
See https://github.com/aptos-labs/aptos-connect/pull/624

Nothing required from dapp developers other than bumping to the latest version of the Wallet Adapter.

https://github.com/user-attachments/assets/86a459ca-9f61-4e4e-ae72-8dc08b305dee

### CODE REVIEW

File: apps/nextjs-example/package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "@radix-ui/react-slot": "^1.0.2",
     "@radix-ui/react-switch": "^1.0.3",
     "@radix-ui/react-toast": "^1.1.5",
+    "@telegram-apps/sdk": "^2.0.0",
```
Comment (0xmaayan) [inline apps/nextjs-example/package.json:33]: In CAD we use '@telegram-apps/sdk-react' - is it a different package?

https://github.com/aptos-labs/create-aptos-dapp/blob/main/templates/clicker-game-tg-mini-app-template/package.json#L31

**Review by 0xmaayan — APPROVED** [review]
LGTM, just a question on the package used here versus CAD

File: apps/nextjs-example/package.json (hunk 1 above)
Comment (hardsetting) [reply apps/nextjs-example/package.json:33]: It's the same underlying package (although I'm using v2 here)

'react-sdk' builds on top of 'sdk'

Comment (hardsetting) [reply apps/nextjs-example/package.json:33]: didn't need full react support

**Review by Joj501 — APPROVED** [review]

**Pull request merged**